sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/SentinelGatewayConstants.java

```diff
@@ -37,9 +37,8 @@
     public static final int URL_MATCH_STRATEGY_REGEX = 2;
 
     public static final int PARAM_MATCH_STRATEGY_EXACT = 0;
-    public static final int PARAM_MATCH_STRATEGY_PREFIX = 1;
```

Comment:
This might bring compatibility problem so we might need to keep its old value.

Reply:
It has been reverted to keep compatibility.